pycarol/data_models/data_models.py

```diff
@@ -352,38 +352,40 @@ def _get_dm_export_stats(self):
         _deprecation_msgs("This function was removed from pyCarol")
         return None
 
-    def _get_min_max(self, query_filter=None):
+    def reprocess_rejected(self, dm_name=None, dm_id=None, delete_records=True):
+        """
+        Reprocess rejected records.
 
-        if query_filter is not None:
-            j = query_filter.type(self.datamodel_name) \
-                .aggregation_list([MINIMUM(name='MINIMUM', params=self.mdm_key), MAXIMUM(name='MAXIMUM',
-                                                                                         params=self.mdm_key)]) \
-                .build().to_json()
-        else:
-            j = Filter.Builder() \
-                .type(self.datamodel_name) \
-                .aggregation_list([MINIMUM(name='MINIMUM', params=self.mdm_key), MAXIMUM(name='MAXIMUM',
-                                                                                         params=self.mdm_key)]) \
-                .build().to_json()
-
-        query = Query(self.carol, index_type=self.index_type, only_hits=False, get_aggs=True, save_results=False,
-                      print_status=True, page_size=0).query(j).go()
-
-        if query.results[0].get('aggs') is None:
-            return None, None
-        min_v = query.results[0]['aggs']['MINIMUM']['value']
-        max_v = query.results[0]['aggs']['MAXIMUM']['value']
-        print(f"Total Hits to reprocess: {query.total_hits}")
-        return min_v, max_v
-
-    def reprocess(self, datamodel_name, n_of_tasks=10, copy_or_move='move', record_type='ALL', query_filter=None):
+        Args:
+            dm_name: `str` default None
+                 Data model name
+            dm_id: `str` default None
+                 Data model ID
+            delete_records: `boll` default True
+                True to delete and reprocess, False for copy and reprocess.
+
+        Returns: dict
+            Task Definition.
+        """
+
+        dm_id = dm_id if dm_id else self.get_by_name(dm_name)['mdmId']
```

Comment:
on Delete operation there is a check and a deprecation message.
I think both should have the same behaviour right?